File: youtubejextractor/src/main/java/com/github/kotvertolet/youtubejextractor/models/test/MiniplayerRenderer.java
package com.github.kotvertolet.youtubejextractor.models.test;

import com.google.gson.annotations.SerializedName;

public class MiniplayerRenderer{

	@SerializedName("playbackMode")
	private String playbackMode;

	public void setPlaybackMode(String playbackMode){
		this.playbackMode = playbackMode;
	}

	public String getPlaybackMode(){
		return playbackMode;
	}
}

File: youtubejextractor/src/main/java/com/github/kotvertolet/youtubejextractor/models/newModels/MiniplayerRenderer.kt
package com.github.kotvertolet.youtubejextractor.models.newModels

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize
import java.io.Serializable

@Parcelize
class MiniplayerRenderer : Parcelable, Serializable {
    @SerializedName("playbackMode")
    var playbackMode: String? = null
    override fun toString(): String {
        return "MiniplayerRenderer{" +
                "playbackMode = '" + playbackMode + '\'' +
                "}"
    }
}